services/uwsgi.ini

```diff
@@ -16,9 +18,26 @@ chmod-socket = 666
 logto = /var/log/uwsgi/%n.log
 log-format = %(addr) - - [%(ltime)] "%(method) %(uri) %(proto)" %(status) %(size) "%(referer)" "%(uagent)" host_hdr=%(host) req_time_elapsed=%(msecs)
 
-processes = 6
+; max number of workers; %k is the number of detected CPU cores
+processes = %(%k + 1)
```

Comment:
The minimum number of workers is 2, the max will be the number of CPU cores plus 1, and at start up the initial number of workers is 3.